### ISSUE TITLE
Windows Configuration Update (KB5035942) installed 7-20 messed up taskbar adn start menus settings

### ISSUE THREAD

**Issue by robtgmann** [valinet/ExplorerPatcher#3496]
Changes that I can notice -- colors of taskbar are lighter.  More important, letters on dark blue background are now black, making them unreadable in task bar app indicators and on app listings in the Start menu.  Backgrounds of tiles in start menu are an ugly gray.  Color changes are so extreme that the taskbar and start menus in Windows 10 mode are virtually useless.

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.  These should be screen shots.  If you can, please change it to english first.

Please include any other software you have installed that modifies the UI of windows.

**Comment by robtgmann** [comment]
WinVer Windows 11 (PRO) Version 23H2 (OS Build 22631.3880)

Explore Patcher version  22621.3527.65.5

You'll find screen shots of my settings in Explorer Patcher by looking at an earlier ticket of mine here.  It's number 3117.  I gave you screen shots which were UNACCEPTABLE on April 14 (same email address for me -- robtgmann@mindspring.com), but gave you acceptable copies on April 15.  I hope you find these useful.  They are what I am currently using.

I don't think I have any other programs that modify Windows UI.

And -- for a little update.  By changing my color scheme I have now have taskbar and menu displays that are legible and satisfactory.  It seems that the Windows update that caused the problem did something to colors.  I had a dark blue background for taskbar and menu which displayed with white letters. The update created a slightly lighter blue, but with black letters that were almost invisible and quite unintelligible. Now I have a slightly lighter blue that displays with white letters which are quite adequate.)

(BTW -- the complaints from issue 3117 remain.  Check for updates, and Update program and restart File Explorer don't work.  Restart File Explorer  does.  I don't need them to work, but thought you might want to know.)

Happy to answer any other questions.  

Best of all things to you!

**Comment by pyrates999** [comment]
Is it this one? https://github.com/valinet/ExplorerPatcher/issues/3117

If so, those screen shots are distorted and cut off.  Please supply the information here so we don't have to hunt for the information requested.

**Comment by robtgmann** [comment]
Scroll down to April 15 input and you will see screen shots of the whole page.  You apparently stopped at the screen shots in the April 14th input from me – the unacceptable ones I mentioned below.  
 
KEEP SCROLLING Just as I told you.
You accepted them last time.  I’m sure they are unchanged from the ones I sent you then – I can see them.

Author: ACT FROM CHOICE 
Read the Kirkus and other reviews   <http://www.actfromchoice.com/sample-reviews-and-press> here
Find it on Amazon and other booksellers      
Please follow me on Twitter: @actfromchoice
And join me on LinkedIn
----------------------------------------------------------------------------------------
Robert Goldmann
Clarity & Performance Coaching
Executives, Professionals, Business owners
www.ClarityAndPerformance.com <http://www.clarityandperformance.com/> 
www.ActFromChoice.com <http://www.actfromchoice.com/> 
510.495.4444
--------------------------------------------------------------------------------------

**Comment by pyrates999** [comment]
Yes, I see it now.  And I said that you should untick register as shell extension.  Then reboot.

You did that and claimed that checking for updates stopped working, etc.

So try the following:

1. in the EP properties application, go to **About** and click on **Restore default settings**.  Then reboot.
2. uninstall EP and see if the issues you mentioned here still occur.
3. do you have any other antivirus installed on your computer besides windows defender?
4. if this is a work computer, the firewall and other network configurations on your work network may be blocking the connection to the internet used for checking for updates.  I would not be installing EP on any work computer where IT can take away your administrative privileges.  And don't assume they won't take that away.  Administrative privileges are required for EP to install, update and be uninstalled.
5. if this is a work computer, do you have crowdstrike installed?

**Comment by robtgmann** [comment]
I apologize for taking so long to get back to you on this.  It got frantic here.
 
Short story is that I am okay with EP as it is, because I am getting the legible display colors I want.
 
But for whatever benefit it might be for you in dealing with this kind of problem.
 
Origin and description of problem:
On July 20 around mid-day, I noticed the colors on my taskbar had changed and it was almost impossible to read the description of open windows: The dark blue background had changed a little, but the letters had changed from white to black.  Shortly afterwards I noticed that the start menu looked awful: previously app tiles and the app listings were invisible or not decipherable because of color changes (though icons were visible).  I’m sorry that I didn’t write down what all the display changes were. 
 
Note that there ware no new installations or restarts I was aware of from opening the computer in the morning to noticing the color changes.
 
Please note that I had purchased this computer, an LG laptop loaded with memory, at the end of November and done nothing I was aware of that would have changed the color settings during the course of that session 7 months later.
 
In researching the problem I discovered that MS had installed a configuration change unbeknown to me (which was noted in my ticket requestion on GitHub.
 
In trying to troubleshoot and make changes I stumbled across a personalization change I could make about colors.  Previously I had light.  I now noticed I had the choice of custom,  I changed to custom, selected the colors I wanted, and am now stable with colors very close to what I want.  The black letters changed to white immediately , and the start menu came back, looking like it did before the weird color changes there. 
 
So I’m happy with what I have an don’t need to spend more time on this. And I can’t reproduce what I had before the color change.
 
Still – to give you the information you requested ….
 
Since I can’t do anything to reproduce the fault, I can’t get any information from doing your suggestions numbered 1 and 2 (below).
 
As to my antivirus – I have Bitdefender security (hate it), not MS defender.  I hope to get rid of it soon.
This is my computer.  There is no work computer.  It’s all on me; I am the one and only administrator, and croudlstrike is not installed.
 
Thanks for all your efforts in trying to fix this.  I’m sorry I can’t contribute any more information that might describe the problem and its cause.  I’m okay with closing this ticket.
 
Best of luck in all things.
 
 

Author: ACT FROM CHOICE 
Read the Kirkus and other reviews   <http://www.actfromchoice.com/sample-reviews-and-press> here
Find it on Amazon and other booksellers      
Please follow me on Twitter: @actfromchoice
And join me on LinkedIn
----------------------------------------------------------------------------------------
Robert Goldmann
Clarity & Performance Coaching
Executives, Professionals, Business owners
 <http://www.clarityandperformance.com/> www.ClarityAndPerformance.com
 <http://www.actfromchoice.com/> www.ActFromChoice.com
510.495.4444
--------------------------------------------------------------------------------------

**Comment by pyrates999** [comment]
My guess is that bitdefender is blocking EP from accessing the internet, thus checking for updates isn't working with it.  Try disabling it and then checking for updates.

Another reason that checking for updates doesn't work is that you need to be on 66.3 pre-release of EP, as previous versions of EP no longer have a working checking for updates feature because there are now 2 exe files instead of 1 since there is now an arm version of EP and an x86-64 version of EP.

As for the colour changes, it looks like you already discovered the reason for it.

**Comment by robtgmann** [comment]
Thanks for the advice. 
 
Alas, removing Bitdefender from my system didn’t help.  I tried to download 66.3, but Defender wouldn’t let me install it.
 
Any tips.  If you tell me to temporarily disable defender to let it install, I’ll do it.
 
Many thanks.
 

Author: ACT FROM CHOICE 
Read the Kirkus and other reviews   <http://www.actfromchoice.com/sample-reviews-and-press> here
Find it on Amazon and other booksellers      
Please follow me on Twitter: @actfromchoice
And join me on LinkedIn
----------------------------------------------------------------------------------------
Robert Goldmann
Clarity & Performance Coaching
Executives, Professionals, Business owners
www.ClarityAndPerformance.com <http://www.clarityandperformance.com/> 
www.ActFromChoice.com <http://www.actfromchoice.com/> 
510.495.4444
--------------------------------------------------------------------------------------

**Comment by pyrates999** [comment]
You can set windows defender to exclude the following:
1. C:\Program Files\ExplorerPatcher
2. %APPDATA%\ExplorerPatcher

Future updates to EP won't be flagged then.

You can also set windows defender to exclude the directory that you manually download EP to so you can install it without windows defender blocking it.

**Comment by robtgmann** [comment]
Okay!  Installed 66.3.
 
Believe it or not, Check for update does not work, nor does update program and restart.  Restart Explorer does.  (Not a problem.  Let’s not torture it any more 😊) 
 
 
 

Author: ACT FROM CHOICE 
Read the Kirkus and other reviews   <http://www.actfromchoice.com/sample-reviews-and-press> here
Find it on Amazon and other booksellers      
Please follow me on Twitter: @actfromchoice
And join me on LinkedIn
----------------------------------------------------------------------------------------
Robert Goldmann
Clarity & Performance Coaching
Executives, Professionals, Business owners
www.ClarityAndPerformance.com <http://www.clarityandperformance.com/> 
www.ActFromChoice.com <http://www.actfromchoice.com/> 
510.495.4444
--------------------------------------------------------------------------------------
